src/main/java/mil/dds/anet/resources/PersonResource.java

```diff
@@ -196,6 +196,25 @@ public Integer updatePerson(@GraphQLRootContext Map<String, Object> context,
     return numRows;
   }
 
+  @GraphQLMutation(name = "updatePersonHistory")
+  public int updatePersonHistory(@GraphQLRootContext Map<String, Object> context,
+      @GraphQLArgument(name = "person") Person p) {
+    final Person user = DaoUtils.getUserFromContext(context);
+    final Person existing = dao.getByUuid(p.getUuid());
+    if (!canCreateOrUpdatePerson(user, existing, false)) {
+      throw new WebApplicationException("You do not have permissions to edit this person",
+          Status.FORBIDDEN);
+    }
+    if (AnetObjectEngine.getInstance().getPersonDao().hasPositionConflict(p)) {
+      throw new WebApplicationException(
+          "There are time conflict between time you selected and selected positions's history ",
```

Comment:
Fix typo's, and the message is also not clear to me. What are the conflicts? I assume some of the history entries on the person passed as input, conflict with the position history of the position in those entries?